README.md

```diff
@@ -1,12 +1,48 @@
 # UA Parser - Browser Extension
 [![All Contributors](https://img.shields.io/badge/all_contributors-3-orange.svg?style=flat-square)](#contributors)
 
+## Prerequisites
```

Comment:
can we move the `Prerequisites` section inside the contribution as it is nothing to do with the actual installation?